### ISSUE TITLE
Add a button in the settings to automatically rename every "KeePassHttp Settings" attributes to "KeePassXC-Browser Settings"

### ISSUE THREAD

**Issue by PlqnK** [keepassxreboot/keepassxc#1591]
First of all thank you all for this 2.3.0 release, I was really looking forward to it and it's really good!

I'm not sure if working on such feature is worth-it because I think by the time you can ship it out most people would have done it manually, but here it goes anyway ¯\_(ツ)_/¯

## Expected Behavior
Have a feature that can automatically rename every "KeePassHttp Settings" labeled attributes in every entries by "KeePassXC-Browser Settings" in order to not have to do it manually.

## Current Behavior
There's no such feature so you either have to rename manually every attributes in every entries or re-validate every entry access when you visite a website after switching to KeePassXC-Browser.

## Possible Solution
Add a button in the section concerning Browser Integration in the settings that will do what I said in the Expected Behavior.

## Context
I just spend 15min renaming them manually one by one.

## Debug Info

KeePassXC - Version 2.3.0
Revision: 4c0ed74

Libraries:
- Qt 5.10.1
- libgcrypt 1.8.2

Operating system: Arch Linux
CPU architecture: x86_64
Kernel: linux 4.15.6-1-ARCH

Enabled extensions:
- Auto-Type
- Browser Integration
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey

**Comment by droidmonkey** [comment]
I think a button in the browser settings view to "Convert legacy browser extension settings" would be very useful. Great suggestion.

**Comment by varjolintu** [comment]
After this PR https://github.com/keepassxreboot/keepassxc/pull/1497 the data will be saved to custom data instead of attributes. So maybe a button that does convert the legacy settings from attributes to custom data would be needed. This would of course convert newer KeePassXC-Browser attributes too.

**Comment by guillaume-uH57J9** [comment]
That's a brilliant idea. 
Is there any workaround besides manually renaming every single entry?

**Comment by varjolintu** [comment]
Nope. Just wait for the update. This should be included in 2.3.2.
